app/src/main/java/com/ivoryworks/cipunchingbag/MainActivity.java

```diff
@@ -1,13 +1,36 @@
 package com.ivoryworks.cipunchingbag;
 
+import android.content.SharedPreferences;
+import android.preference.PreferenceManager;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.widget.TextView;
 
 public class MainActivity extends AppCompatActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+        Integer i = null;
+        i.toString();
+
+        SharedPreferences.Editor editor = PreferenceManager.getDefaultSharedPreferences(this).edit();
+        editor.commit();
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ivoryworks/CI-PunchingBag/blob/ab24793045a516fa587e5993e796c4fedd7e27e3/app/src/main/java/com/ivoryworks/cipunchingbag/MainActivity.java#L20"></span>Consider using <code>apply()</code> instead; <code>commit</code> writes its data to persistent storage immediately, whereas <code>apply</code> will handle it in the background</td>
    </tr>
  </tbody>
</table>